flink-tests/pom.xml

```diff
@@ -54,6 +54,12 @@ under the License.
 
 		<dependency>
 			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-shaded-hadoop2</artifactId>
+			<version>${project.version}</version>
+		</dependency>
```

Comment:
add test scope?

Reply:
I'll add test scope and see if everything still runs.